### ISSUE TITLE
How get 2 side image?

### ISSUE THREAD

**Issue by ulises58** [Luomusha/node-twain#8]
Since I can get the image from both sides, it usually only lets me have one side of the image, but I don't know how to tell it to output both.

**Comment by Luomusha** [comment]
According to the twain-specification, you need to set Capability : **CAP_DUPLEX**

This is how the method should be called
```
session.setCapability(CAP_DUPLEX,  CAP_DUPLEXENABLED)
```

However, this method “setCapability” has not been implemented yet.
I don't have a scanner now.

**Comment by ulises58** [comment]
Certainly! Here's a question you can use to inquire about the origin of the session variable in your code:

"In the provided code using the node-twain library, there is a reference to session.setCallback(). However, I haven't explicitly defined the session variable in my code. Could you please clarify where the session variable is supposed to come from or how it should be created before using setCallback()?"

**Comment by marchbold** [comment]
Any plans to implement `setCapability`?
